src/api/embed_helpers.cc

```diff
@@ -129,12 +130,21 @@ CommonEnvironmentSetup::CommonEnvironmentSetup(
   {
     Locker locker(isolate);
     Isolate::Scope isolate_scope(isolate);
+    HandleScope handle_scope(isolate);
+
+    TryCatch bootstrapCatch(isolate);
+    auto print_Exception = OnScopeLeave([&]() {
+      if (bootstrapCatch.HasCaught()) {
```

Comment:
Should this check `CanContinue()` also?

Reply:
I think the code below should return early when there is an exception, so it's not necessary to check here because what we do should not result in JS execution anyway, just communicating the stack trace back to the caller.